static/sass/styles.scss

```diff
@@ -1099,3 +1099,10 @@ hr.p-separator.is-shallow {
 .no-border::after {
   content: none !important;
 }
+
+.u-label-sizing {
```

Comment:
Could you propose this to vanilla please?

Reply:
Looks like this is used once. Are you sure we need a new utility for a single case?

Reply:
It came up very early and I thought it may be repeated. I will remove it in place of inline styling

Reply:
The styling is still present?

Reply:
Sorry, I have updated that now